src/ui/design/Button/Button.stories.tsx

```diff
@@ -11,7 +12,7 @@ export default {
 };
 
 const Template: ComponentStory<typeof Button> = (args) => (
-  <Button {...args}>Button</Button>
+  <Button {...args}>{text("label", "Button")}</Button>
```

Comment:
the `text(...)` will show up in "Knobs" tab:
![image](https://user-images.githubusercontent.com/7823011/206701731-ad62a5d7-457a-4365-8a14-d9f67fbb6f9a.png)